app/assets/stylesheets/_commons.css.scss

```diff
@@ -76,7 +64,7 @@ body {
     tr td, tr th { color: $light-gray !important; }
 
     td:before {
-      background-color: #5AF2AC !important;
+      background-color: #2CDF79 !important;
```

Comment:
Color literals like `#2CDF79` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#2CDF79` should be written as `#2cdf79`<br>!important should not be used